lib/sender.js

```diff
@@ -115,11 +115,15 @@ function send(client) {
 
             request(req, (err, response, body) => {
                 if (err) {
-                    return reject(`Problem with Request: ${err.message}`);
+                    return reject(
+                        new Error(`Problem with UPS Request: ${err.message}`)
+                    );
                 }
 
                 if (response.statusCode !== 200) {
-                    return reject(response.statusCode);
+                    return reject(
+                        new Error(`UPS request returned status code: ${response.statusCode}`)
```

Comment:
I'm not 100% sure on that either. I've only logged one once, and it was empty IIRC. Not sure about other errors.